src/main/java/se/walkercrou/places/GooglePlaces.java

```diff
@@ -255,7 +253,7 @@ public GooglePlaces(String apiKey, RequestHandler requestHandler) {
      * @param apiKey that has been registered on the Google Developer Console
      */
     public GooglePlaces(String apiKey) {
-        this(apiKey, new DefaultRequestHandler());
+        this(apiKey, new GAERequestHandler());
     }
```

Comment:
Hello, I was thinking of making the default constructor to be DefaultRequestHandler() however, I have gone and customized your library for GoogleAppEngine which means that I have used classes such as com.google.appengine.api.images.Image and import com.google.appengine.api.images.ImagesServiceFactory to replace your awt.images code - see changes made in the Photo.java class. 
A person who wishes to use this library on GAE can therefore not easy just substitute their requestHandler for the GAERequestHandler and make it work on GAE....Would you be able to pull this request to a new branch on your side for a separate library just for GAE?